packages/components/toggle/src/toggle.types.ts

```diff
@@ -1,30 +1,27 @@
-import {
-  ChangeEventHandler,
-  HTMLAttributes,
-  InputHTMLAttributes,
-  LabelHTMLAttributes,
-} from 'react';
+import { ChangeEventHandler, HTMLAttributes, LabelHTMLAttributes } from 'react';
+
+export type IconNames = 'star' | 'chevron-down' | 'chevron-up' | 'close';
 
 export interface ToggleProps extends HTMLAttributes<HTMLDivElement> {
   disabled?: boolean;
   checked?: boolean;
+  label?: string;
```

Comment:
you can remove label, we don't need it

Reply:
We need to have the label to fulfill the accessibility requests